src/main/markers/waveform.segments.js

```diff
@@ -25,10 +25,10 @@ define([
       return view;
     });
 
+    var segmentId = 0;
     var createSegmentWaveform = function (options) {
-      var segmentId = "segment" + self.segments.length;
       var segment = {
-        id: segmentId,
+        id: options.id || ('peaks.segment.' + segmentId++),
```

Comment:
A minor nitpick, but I think we should allow zero as a valid ID, and only set a default ID if undefined or null.